queue_services/business-emailer/.env.sample

```diff
@@ -1,48 +1,48 @@
 # Flask
 FLASK_ENV=
 
-ACCOUNT_SVC_AUTH_URL=
-ACCOUNT_SVC_CLIENT_ID=
-ACCOUNT_SVC_CLIENT_SECRET=
-APP_FILE=
-AUTH_URL=
-DASHBOARD_URL=
-
+LD_SDK_KEY=
+KEYCLOAK_AUTH_TOKEN_URL=
+ENTITY_SERVICE_ACCOUNT_CLIENT_ID=
+ENTITY_SERVICE_ACCOUNT_CLIENT_SECRET=
 
-LEGAL_API_URL=
-#LEGAL_API_URL=
+AUTH_API_URL=
+AUTH_API_VERSION=
 NOTIFY_API_URL=
-#NOTIFY_API_URL=
+NOTIFY_API_VERSION=
+BUSINESS_API_URL=
+BUSINESS_API_VERSION2=
```

Comment:
One last thing, LEGAL_API_URL and LEGAL_API_VERSION_2 are the values that exist in 1pass so if you're going to update it it should be to those (this is what it'll need to be altered to when we hook up the automated CD)